ipaserver/plugins/session.py

```diff
@@ -21,7 +20,9 @@ def execute(self, *args, **options):
         ccache_name = getattr(context, 'ccache_name', None)
         if ccache_name is None:
             self.debug('session logout command: no ccache_name found')
+        else:
+            delattr(context, 'ccache_name')
 
-        logout(ccache_name)
+        setattr(context, 'logout_cookie', '')
```

Comment:
for consistency, all manipulations of context happen via setattr/getattr/delattr